### DISCUSSION TITLE
Welcome to hirob-em-pcbdesign Discussions!

### DISCUSSION THREAD

**Discussion by husamettinarabaci** [Ubn-Jr/hirob-em-pcbdesign#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yusufyilmazf** [comment]
### Hello, I'm from the hardware team @yusufyilmazf 
  I am responsible for drawing electronic circuit boards in the [Hi-Rob](https://github.com/orgs/Ubn-Jr/projects/4) project. The electronic circuit board forms the basis of the hardware of our project. This board contains the electronic components of our project and determines how they will work together. When designing the circuit board, I use Altium Designer as a circuit design application. Before designing the circuit with Altium Designer, [we](https://github.com/orgs/Ubn-Jr/teams/hirob-em) chose the appropriate motor driver, motor, microprocessor etc. for the Hi-Rob. and we are researching suitable power supply for these components. In order to create a stable circuit as a result of the research, we prepare a test card with the selected components and carry out the tests with the [hirob-em-embedded](https://github.com/Ubn-Jr/hirob-em-embedded) team.
